lib/Sema/CSSimplify.cpp

```diff
@@ -2161,22 +2161,37 @@ static ConstraintFix *fixPropertyWrapperFailure(
     }
   };
 
+  // Local function to substitute into the
```

Comment:
Missing the rest of the comment :) It would be great if the comment explained why we have to do that for posterity.

Reply:
Yeah, I'll finish the comment. It's because we're getting an interface type for a member, but we need to substitute the base type into the member type for it to make sense.

Reply:
Interesting, we don’t cover this case at the moment, only if it’s a member reference.

Reply:
Oh, it looks like I misunderstood what type `baseTy` was meaning to use. I have a better fix coming